### PR TITLE
TAXI-16-MD make_dataset.py into class

### PR DESCRIPTION
**Pull request by AdamJedz**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -45,13 +40,22 @@ def run_predict_model():
     parser.add_argument("--train_model", type=bool)
     parser.add_argument("--predict_model", type=bool)
     args = parser.parse_args()
+
+    dataset_maker = DatasetMaker(
+        input_file_name="final_taxi_data",
+        test_size=0.3,
+        save_files=True,
+        output_train_test_file_names=("train_data", "test_data"),
+        output_cleaned_file_name="train_data_clean",
```
Comment (dk1000) [inline main.py:49]: I thought we agreed, to choose dict's one key into each class. I would expect here sth like params['dataset_params']

File: main.py [hunk 2]
```diff
@@ -45,13 +40,22 @@ def run_predict_model():
...
+    )
+
     if args.make_dataset is True:
```
Comment (dk1000) [inline main.py:52]: can't it be just `if args.make_dataset:`

File: main.py (hunk 1 above)
Comment (AdamJedz) [reply main.py:49]: I thought to leave it as it is until we have a config file. But, you're right - I will create temporary dict in main.py to store all the parameters for now

File: main.py (hunk 2 above)
Comment (AdamJedz) [reply main.py:52]: True

**Review by dk1000 — CHANGES_REQUESTED** [review]
Moreover, I believe it's time to add yaml as well, if done later, it will cause need of editing currently provided changes (the same work done twice).

File: main.py (hunk 1 above)
Comment (dk1000) [reply main.py:49]: so comes a question if config to be done now, I believe so. But it adds bigger chunk of code in one PR

**Comment by AdamJedz** [comment]
> Moreover, I believe it's time to add yaml as well, if done later, it will cause need of editing currently provided changes (the same work done twice).

So it will be the next task I'll take after that one

**Review by dk1000 — APPROVED** [review]
ładnie

**Pull request merged**